src/Collection/Collection.php

```diff
@@ -14,11 +14,15 @@ class Collection implements Countable, IteratorAggregate
 {
     protected $values = [];
 
-    public function all()
+    public function all(): array
     {
         return $this->values;
     }
 
+    /**
+     * @return mixed|void
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint
```

Comment:
Is there a way to opt-out from this check? This looks like very verbose stuff.

Reply:
Given [docs](https://github.com/slevomat/coding-standard#slevomatcodingstandardtypehintsreturntypehint-) none that I am aware of.

Reply:
I found a way. Please, drop those lines.

Reply:
Will do. Can you share details how you're doing that?

Reply:
Ok, I see how you've added configuration setting to phpcs.xml. Didn't know about that one :+1: